doc/releases/migration-guide-4.1.rst

```diff
@@ -408,6 +408,12 @@ Bluetooth Host
   makes a device vulnerable for downgrade attacks. If an application still needs to use LE legacy
   pairing, it should disable :kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY` manually.
 
+* The ``CONFIG_BT_SEND_ECC_EMULATION`` Kconfig option has been removed, and the host will now
+  perform ECDH operations directly itself by calling PSA APIs.
+
+* The prompt for :kconfig:option:`CONFIG_BT_ECC` has been removed, since it only offers an internal
+  API, meaning internal users should explicitly select it in their respective Kconfig options.
```

Comment:
I see that the above items doesn't do this, but it's a fairly common practice to also add a reference to the PR making this change. It's often helpful for people doing the migration to see how it was done in e.g. samples etc.